Add waffle tutorial

yivlad opened the pull request:
**Description**
Add [Waffle](https://github.com/TrueFiEng/Waffle)  tutorial to `getting-started` folder.

qbzzt commented:
Thank you. I'm trying to make it so it'll work without requiring you to install a devnode locally.

So far I got:

```
describe('Optimism', () => {
  const provider = new OptimismProvider('https://kovan.optimism.io');
  let wallet: Wallet;
  let contract: MockContract;

  before(async () => {
    // const wallets = provider.getWallets();
    // wallet = wallets[0];
    wallet = Wallet.fromMnemonic(<valid mnemonic goes here>).connect(provider)
  });
```

And to account for the fact it's a lot slower to do over a network:
```
yarn test --timeout 60000
```

But now I'm getting this error:
```
  Optimism
    ✔ Emits One (6166ms)
    1) Reverts


  1 passing (15s)
  1 failing

  1) Optimism
       Reverts:
     AssertionError: Expected transaction to be reverted with "Revert cause", but other reason was found: "Revert cause"},"id"
```

Any idea what's the difference?

yivlad commented:
@qbzzt the problem was on the Waffle's side. Fixed, released a new version and updated here. Also modified tutorial to use Kovan test network by default. Should be all good now.

qbzzt reviewed (approved):
Thank you. It's ready.
